manual/src/threelang/tfgrid/vm.md

```diff
@@ -0,0 +1,53 @@
+# VM Namespace
+
+- To deploy a number of virtual machines on the same network, use the vm namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.vm.create
+- parameters:
+  - network [required]
+  - farm_id [optional]
+    - if 0, the grid nodes will be chosen at random and the deployed vms might span multiple farms.
+  - times [required]
+    - indicates how many machines to deploy
+    - a number in the range [1, 252]
+  - capacity [required]
+    - a string in ['small', 'medium', 'large', 'extra-large'] indicating the capacity of the machines
+    - small: 1 vCPU, 2GB RAM, 10GB SSD
+    - medium: 2 vCPU, 4GB RAM, 50GB SSD
+    - large: 4 vCPU, 8GB RAM, 240 SSD
+    - extra-large: 8vCPU, 16GB RAM, 480GB SSD
+  - disk_size [optional]
+    - size in GB of disk to be mounted on each machine at "/mnt/disk"
+  - ssh_key [required]
+- arguments:
+  - add_wireguard_access
+  - public_ips
```

Comment:
This means adding a public_ip to the vm, right? Rename to add_public_ip in that case